src/Uno.AzureDevOps/Uno.AzureDevOps.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.nventive.uno.azuredevops" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="16" android:targetSdkVersion="28" />
-	<application android:label="uado" android:icon="@mipmap/icon" android:allowBackup="false"></application>
```

Comment:
Why remove this?

Reply:
Not intended, seems manifest permission in VS removed it :O